stant-album/ViewController.swift

```diff
@@ -44,46 +44,63 @@ class ViewController: UIViewController {
     
     override func viewDidAppear(_ animated: Bool) {
         gradient.frame = self.view.safeAreaLayoutGuide.layoutFrame
-        child.view.frame = self.view.safeAreaLayoutGuide.layoutFrame
+        loadSpinner.view.frame = self.view.safeAreaLayoutGuide.layoutFrame
     }
 
     @IBAction func getNewAlbumAction(_ sender: Any) {
         self.getNewAlbum()
     }
     
+    func makeAlert(errorTitle: String, errorMessage: String) {
+        let alert = UIAlertController(title: errorTitle, message: errorMessage, preferredStyle: UIAlertController.Style.alert)
+        alert.addAction(UIAlertAction(title: "OK", style: UIAlertAction.Style.default, handler: nil))
+        self.present(alert, animated: true, completion: nil)
+    }
+    
     func getNewAlbum() {
-        let albumIndex      = Int.random(in: 0..<albumGenerator.albumList.count)
-        let newAlbum: Album = albumGenerator.albumList[albumIndex]
-        self.updateFields(newAlbum)
+        makeRequest(getFromURL: "https://fakerapiexample.herokuapp.com/album", requestCompletionHandler: { data, error in
```

Comment:
Do you remember what we talked about separate responsibilities? 
Try to put it in another class (like a presenter, or just an api data source) and make view controller haddler only view stuffs (what happens when it is success or failure).

Reply:
Done